### ISSUE TITLE
Synchronization of the 'item list' and 'focus bar'

### ISSUE THREAD

**Issue by Paskudlo** [flowkeeper-org/fk-desktop#54]
Hello,

I tend to work somewhat chaotically, jumping from one task to another. To facilitate this workflow, I create many tasks beforehand and then switch between them after a few pomodoro timer instances.

The issue with this kind of workflow is that after a few switches, I've noticed that the 'next pomodoro' play button on the focus bar starts the timer only on the task I initially selected, not the one I've selected in the 'item list'. This has led to some pomodoro timer mismatches, causing me to create additional pomodoros and manage the task list unnecessarily.

Question: Is this a necessary behavior of the 'focus bar'? If not, could you make it so that the bar syncs with the selected task in the 'item list'? (So, selecting a new item in the list and pressing the 'next pomodoro' button on the focus bar would be equivalent to selecting a new item in the list and using the ctrl+s hotkey.)

Thanks!

**Comment by co-stig** [comment]
Hi there,

Thanks for the idea! Let me know what you think about this -- for v0.9.0 I was going to streamline the Focus bar by moving the icons under the context menu: 

<img width="1141" alt="Screenshot 2024-12-08 at 11 38 48" src="https://github.com/user-attachments/assets/2d06509b-512b-4b00-a908-08bfc8cc57d7">

Note the "Next Pomodoro" item -- it will contain the work item name, so that it's harder to make a mistake. I can also add the second menu item like "Next Pomodoro (<selected item title>)", but not sure it'll be clear for the user.

/ Contantine

**Comment by Paskudlo** [comment]
Hi again,

I think this version looks clean, and some users might prefer this kind of interface change.

Personally, I prefer having a standalone ▶ "Next Pomodoro" button as it is, because it requires fewer clicks. (Your solution essentially doubles the number of clicks users have to make during their work sessions.) 
Here's how I use the app: I keep Flowkeeper on my second monitor, in the far left corner. When I'm focused on a task that takes multiple Pomodoro timers, I just click on the ▶ button on the focus bar once the rest period is over to immediately resume working.

So, my workflow looks like this:

1. Select a task from the item list.
2. Press the ▶ button on the focus bar to start a Pomodoro timer.
3. Move the pinned focus bar to the far left corner.
4. Work until the rest timer goes off.
5. Relax to the beautiful piano music for 5 minutes.
6. Press the ▶ button on the focus bar again to resume the work.

What I would like to have is the ability to switch tasks seamlessly in this workflow. So, the same loop overall, but after relaxing during the rest timer music:

1. Drag the Flowkeeper window up and select a new task in the "item list."
2. Press the ▶ button on the focus bar to start a Pomodoro timer for that new task.
3. Drag the Flowkeeper window back to the left corner.
4. Work until the timer goes off.
5. Repeat the previous loop.

Sorry for the detailed explanation. Thanks for sharing your ideas for the update!

**Comment by co-stig** [comment]
This is great, I really appreciate the level of details you provided there, it's exactly the kind of feedback I am looking for! I'll consider it and will try to implement something that accommodates your workflow. In the meantime, you can use the toolbar button to start the selected item:
<img width="622" alt="Screenshot 2024-12-08 at 17 11 10" src="https://github.com/user-attachments/assets/e23132dc-f524-4fd3-9406-c9a375f7b1c5">

**Comment by Paskudlo** [comment]
Yes, that's exactly the workaround I use right now! I'm glad my feedback was helpful.
